src/components/AppToolbar.tsx

```diff
@@ -216,8 +224,8 @@ class AppToolbarInternal extends React.Component<AppToolbarInternalProps> {
             <IconText>{t("Open")}</IconText>
           </ToolbarAction>
           <ToolbarAction wdKey="nav:export" onClick={this.props.onToggleModal.bind(this, 'export')}>
-            <MdFileDownload />
-            <IconText>{t("Export")}</IconText>
+            <MdSave />
```

Comment:
Shouldn't the save be available only if I opened a file from my machine? What happens when I set a URL or choose an existing style?

Reply:
I had the same thoughts to have sometimes the label "download" and sometimes "save" while implementation. But I ended up using only "save" because there is no difference in where the style was loaded from. If a file handle exists, it will use it, it it doesn't it prompts the user to create a new. 
I thought it could be confusing to the user if the labels would change. Thinking of it, the "actual" download process would be when the user loads a style from an url. 😄